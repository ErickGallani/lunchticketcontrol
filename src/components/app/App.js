import React from 'react';
import PropTypes from 'prop-types';
import './app.css';

export default class App extends React.Component {
    constructor(props){
        super(props);
    }
    
    render() {      
      return (
        <div className="app">
            <p>Test</p>
        </div>
      );
    }
  }
  
  