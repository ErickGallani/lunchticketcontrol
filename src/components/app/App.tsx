import * as React from 'react';
import * as PropTypes from 'prop-types';
import './app.css';

export default class App extends React.Component {
    constructor(props: any){
        super(props);
    }
    
    render() {      
      return (
        <div className="app">
            <p>Test</p>
        </div>
      );
    }
  }
  
  