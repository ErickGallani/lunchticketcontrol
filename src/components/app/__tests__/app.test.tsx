import * as React from 'react';
import App from "../App";
import { shallow, mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders correctly app', () => {
  const elem = shallow(<App />);
  expect(shallowToJson(elem)).toMatchSnapshot();
});