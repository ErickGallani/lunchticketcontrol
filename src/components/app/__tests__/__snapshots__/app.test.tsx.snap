// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly app 1`] = `
<div
  className="app"
>
  <p>
    Test
  </p>
</div>
`;
